src/GenericPhotosensor.cpp

```diff
@@ -96,9 +96,46 @@ void GenericPhotosensor::Construct()
 
   name = "PHOTOSENSOR_OPSURF";
 
-  G4double energy[]       = {0.2*eV, 11.5*eV};
-  G4double reflectivity[] = {0.0   ,  0.0};
-  G4double efficiency[]   = {1.0   ,  1.0};
+  G4double energy[]       = {11.500*eV, 3.875*eV, 3.836*eV, 3.815*eV, 3.794*eV, 3.773*eV, 
```

Comment:
Let's use here the constants that are defined in **OpticalMaterialProperties**:
`OpticalMaterialProperties::energy_max` and `OpticalMaterialProperties::energy_min`

https://github.com/jmalbos/G4OpSim/blob/1765f821b924479352946e3a969d205cd1b4269c/src/OpticalMaterialProperties.h#L19